filter.js

```diff
@@ -45,3 +45,4 @@ function filterData (rawData, excludes) {
 }
 
 module.exports.filterData = filterData
+module.exports.isExcluded = isExcluded
```

Comment:
We're only exporting this so that the tests can see it to test, right? Is there a way we can export this function _only_ to the tests?

Reply:
I made small research and I don't see a native way in Node.js to export one function only for one file.
I only saw this library which I am not sure how will help us with this: 
https://github.com/jhnns/rewire

Also, I am not sure is it worth it to add a new dependency only because of this.

Reply:
Agreed, it's not worth adding a dependency for this.